### PR TITLE
Exclude seeds from backup

### PR DESCRIPTION
**Pull request by rorp**

### CODE REVIEW

File: app/server/src/main/scala/org/bitcoins/server/BitcoinSAppConfig.scala [hunk 1]
```diff
@@ -259,21 +261,30 @@ object BitcoinSAppConfig extends Logging {
     conf.bitcoindRpcConf
   }
 
-  def zipDatadir(source: Path, target: Path): Path = {
+  def zipDatadir(source: Path, target: Path): Try[Path] = Try {
```
Comment (Christewart) [inline app/server/src/main/scala/org/bitcoins/server/BitcoinSAppConfig.scala:264]: Should we have an option to include the seed? Thoughts @user411 ?

Comment (user411) [reply app/server/src/main/scala/org/bitcoins/server/BitcoinSAppConfig.scala:264]: It seems like there may be times where people want to pull a backup from Umbrel for use on desktop. Would extracting the seed damage the security of the Umbrel install at all?

Comment (rorp) [reply app/server/src/main/scala/org/bitcoins/server/BitcoinSAppConfig.scala:264]: Extracting the seed could damage security of the wallet: one should never have an electronic copy of the seed. The users should be able to write down the seed instead.

Comment (Christewart) [reply app/server/src/main/scala/org/bitcoins/server/BitcoinSAppConfig.scala:264]: > It seems like there may be times where people want to pull a backup from Umbrel for use on desktop. Would extracting the seed damage the security of the Umbrel install at all?

I believe the umbrel environment derives a seed for us based off of the root seed. I don't believe it is possible to reverse engineer the root umbrel seed from a specific app's seed.

Comment (Christewart) [reply app/server/src/main/scala/org/bitcoins/server/BitcoinSAppConfig.scala:264]: > Extracting the seed could damage security of the wallet: one should never have an electronic copy of the seed. The users should be able to write down the seed instead.

What if the seed is encrypted? We support encrypted seeds in the `seeds/` directory.

**Pull request merged**